Src/Base/AMReX_ParallelDescriptor.cpp

```diff
@@ -339,6 +355,15 @@ ParallelDescriptor::StartParallel (int*    argc,
     if (mpi_version < 3) amrex::Abort("MPI 3 is needed because USE_MPI3=TRUE");
 #endif
 
+#ifdef AMREX_MPI_MULTIPLE
+    int provided = -1;
+    MPI_Query_thread(&provided);
+    amrex::Print() << "MPI Thread Initialized with support level " << provided << "." << std::endl;
+
+#else
+    amrex::Print() << "MPI Initialized." << std::endl;
+#endif
+
```

Comment:
In `amrex::Initialize`, we already print mpi information.  So maybe this can be merged into there.  And it should not print unless verbosity is on for compliance with XSDK.